torch/onnx/utils.py

```diff
@@ -1861,6 +1880,7 @@ def _run_symbolic_function(
         if symbolic_function_group is not None:
             symbolic_fn = symbolic_function_group.get(opset_version)
             if symbolic_fn is not None:
+                # TODO(justinchuby): Wrap almost identical attrs assignment or comment the difference.
```

Comment:
Hi! I don't have context to this TODO and thus am not fitting to be the contact person. I suggest creating an issue for tracking, or alternatively using your github alias.

Reply:
That said I am happy to do the actual work of it. It's just that whatever alias written here needs to be where the information is, as a contact.

Reply:
No worries, as the author of the [new registry](https://github.com/pytorch/pytorch/pull/84382) who heavily reimplemented/refactored this code, I was expecting you to be the most fitting and own it. Nonetheless, I've unassigned the comment.